<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MeasGeomExportDialog</class>
 <widget class="QDialog" name="MeasGeomExportDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>340</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Export Measurement to Geometry</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="export_type_label">
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>30</height>
        </size>
       </property>
       <property name="font">
        <font>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="text">
        <string>Export Type :</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="shapefile_btn">
       <property name="text">
        <string>Shapefile</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QRadioButton" name="ASCII_btn">
         <property name="text">
          <string>ASCII : </string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="xyz_btn">
         <property name="text">
          <string>xyz</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="lat_lon_btn">
         <property name="text">
          <string>lon/lat</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="Line" name="separtor">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="data_type_label">
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>30</height>
        </size>
       </property>
       <property name="font">
        <font>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="text">
        <string>Data to export :</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="points_checkBox">
       <property name="text">
        <string>Points</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="lines_checkBox">
       <property name="text">
        <string>Lines</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="areas_checkBox">
       <property name="text">
        <string>Areas</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="Line" name="separator2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="export_file_label">
       <property name="text">
        <string>Export File :</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLineEdit" name="path_lineEdit"/>
       </item>
       <item>
        <widget class="QPushButton" name="path_pushButton">
         <property name="text">
          <string>choose path</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QDialogButtonBox" name="valid_cancel_buttonBox">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="standardButtons">
        <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>valid_cancel_buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>MeasGeomExportDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>valid_cancel_buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>MeasGeomExportDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
